src/server/graphql/queries/isBillingLeader.js

```diff
@@ -10,7 +10,9 @@ export default {
   },
   type: GraphQLBoolean,
   description: 'true if the user is a part of the supplied orgId',
-  resolve: (source, {orgId}) => {
-    return Boolean(source.userOrgs.find((userOrg) => userOrg.id === orgId && userOrg.role === BILLING_LEADER));
+  resolve: async (source, {orgId}, {dataLoader}) => {
```

Comment:
Seems like we should be doing access control here.

Reply:
Oooo this is where it gets fun in graphql land. we handle this in the parent. so the logic is, "whoever has access to the user can see if that user is a billing leader". since we're getting the userId from the source, we know they already must have access to that user, so they're safe